src/index.js

```diff
@@ -216,6 +216,9 @@ async function executeActions(req, context, params) {
       log.error(`no valid response could be fetched: ${e}`);
       return new Response('', {
         status: /* istanbul ignore next */ e.statusCode === 502 ? 504 : e.statusCode,
+        headers: {
+          'x-error': e.message,
```

Comment:
shoulnd't we try to aggregate the errors from the down-stream errors?

Reply:
`helix-dispatch` is failing fast, so the first error will get propagated, even if there are more than one. I don't think it is worth to wait for the remaining errors, especially when this gets us close to the 30 second limit.

Reply:
no, I mean, aggregate the `x-error` headers from the downstream actions. for example, if `static` returns with a `500` and has a `x-error` header, then combine it here.